Client/Frontend/Browser/Helpers/LaunchHelper.swift

```diff
@@ -64,4 +94,16 @@ public actor LaunchHelper {
       await AdBlockEngineManager.shared.startTimer()
     }
   }
+  
+  private func measuredTask(label: String, callback: () async -> Void) async {
```

Comment:
You probably want `@Sendable` on the callback (i.e. `callback: @Sendable () async -> Void`)

Reply:
I will remove this logging for now and will wait for you to get back to get this done right.